server.js

```diff
@@ -36,20 +36,16 @@ MongoDB Connectors/Collection creation
 */
 
 const MongoClient = require('mongodb').MongoClient;
-const dbName = 'restaurants-web-api-db'
-const dbHost = `protainer.lan:27017`
-const url = `mongodb://${dbHost}/${dbName}`;
 
-MongoClient.connect(url, function(err, db) {
+MongoClient.connect(dbConfig.url, function(err, db) {
```

Comment:
So there is a lot of code in this file now. As I imagine you want to add more features, now is probably a good time to start separating our concerns and splitting things out into different classes/files before the project gets bigger. One good place to start might be the code to access the database. At the minute we have one collection called recentlocations, so maybe create a file called recentlocationsrepository, and in here we can have a mongo client, then split out each db call into a function that we can import for use in another file. Something a bit like:
```
const MongoClient = require('mongodb').MongoClient;

function checkMongoClientConnection() {
  MongoClient.connect(dbConfig.url, function(err, db) {
    if (err) console.error(err);
    db.close();
  });
}

function initialiseRecentLocationsCollection() {
  MongoClient.connect(dbConfig.url, function(err, db) {
      if (err) console.error(err);
      let dbo = db.db(dbConfig.dbName);
      dbo.createCollection("recentlocations", function(err, res) {
          db.close();
      });
  });
}
```

Reply:
Please see InitaliseMongoCollections.js 

I've split that into their own file, could go further but that seems enough for now. Probably will look at turning other other 2 files into Classes, let me know what you think.